### ISSUE TITLE
Chat Channel changes, continued

### ISSUE THREAD

**Issue by mootkins** [aaronma37/Deathlog#61]
Updated:
Updated to latest via github, still puts deathlog channel at 1, have to manually move to lower. Does not save channel locations.

Still not working, opening new issue in regards to #51 

Original Post:

Can we get deathlog to stop changing its chat tab position to 1? I want local to be 1, trade to be 2, but every single session it gets changed for hcdeathalertschannel to be 1, and pushing the others aside.

I then have to go and manually set it to 7/8, but itll revert if i ever log out.

![image](https://github.com/aaronma37/Deathlog/assets/54551598/fa4a414d-9372-44b5-85c7-2ac1be9caac6)
The above is what is happening.

![image](https://github.com/aaronma37/Deathlog/assets/54551598/4db5fd37-df33-48cf-8364-a0ee0c9ac530)
This is how I manually set it every session

**Comment by aaronma37** [comment]
Have you tried /resetchat or editing the file in WTF > Account > # > Server > Character folder to reorder the channel numbers.

It would look like:  
General 0 1 0  
Trade 0 2 0  
LocalDefense 0 3 0  
LookingForGroup 0 4 0

**Comment by aaronma37** [comment]
Have you tried /resetchat or editing the file in WTF > Account > # > Server > Character folder to reorder the channel numbers.

It would look like:  
General 0 1 0  
Trade 0 2 0  
LocalDefense 0 3 0  
LookingForGroup 0 4 0

**Comment by mootkins** [comment]
I will try these out and give you an update in the next couple minutes

**Comment by mootkins** [comment]
Mine looks like this:

|CHANNELS
1 0 1 1
2 0 2 1
22 0 3 1
24 1 4 1
LFG 1 5 3
23 1 6 1
aotc 1 7 3
END

Any input on what to do?

**Comment by mootkins** [comment]
/resetchat only reset font, size, window settings, not any channels

**Comment by mootkins** [comment]
I figured it out, uninstalling the addon "Tombstones" and "Autolayer" no longer makes hcdeathalerts move to the top

**Comment by aaronma37** [comment]
Its probably related to tombstones, they hook into the same channel

**Comment by mootkins** [comment]
Yeah most recent update has broken it again, no longer have tombstones and
it still moves death alert channel to slot 1
